src/adapters/routes/userRutes.ts

```diff
@@ -0,0 +1,20 @@
+import { Router, Request, Response } from 'express';
+import { UserService } from '../../domain/services/userService';
+import { MongoUserRepository } from '../../adapters/repositories/userRepo';
+
+const router = Router();
+const userRepository = new MongoUserRepository(); 
+const userService = new UserService(userRepository); 
+
+router.post('/register', async (req: Request, res: Response) => {
```

Comment:
You should create a userController.ts in infrastructure to manage the logic of a controller. This file only contains a relationship between routes and controller methods.